db/migrations/20220603140800_add_table_users.go

```diff
@@ -0,0 +1,49 @@
+package migrations
+
+import (
+	_ "embed"
+	"time"
+
+	logging "github.com/RedHatInsights/sources-api-go/logger"
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+func AddTableUsers() *gormigrate.Migration {
+	type Tenant struct {
+		Id int64
+	}
+
+	type User struct {
+		Id     int64  `gorm:"primarykey"`
```

Comment:
I think this needs the `autoIncrement` tag too, right?

Reply:
gorm do this automatically for numeric primary keys

```
sources_api_development=# \d public.users
                                            Table "public.users"
   Column   |            Type             | Collation | Nullable |                 Default
------------+-----------------------------+-----------+----------+------------------------------------------
 id         | bigint                      |           | not null | nextval('public.users_id_seq'::regclass)
 user_id    | text                        |           | not null |
 tenant_id  | bigint                      |           | not null |
 created_at | timestamp without time zone |           | not null |
 updated_at | timestamp without time zone |           | not null |
Indexes:
    "users_pkey" PRIMARY KEY, btree (id)
    "users_user_id_idx" UNIQUE, btree (user_id)
    "users_user_id_tenant_id_idx" UNIQUE, btree (user_id, tenant_id)
Foreign-key constraints:
    "fk_users_tenant" FOREIGN KEY (tenant_id) REFERENCES public.tenants(id)
```

Reply:
Aaaah great then!